server/src/main/java/org/exoplatform/chat/server/ChatServer.java

```
@@ -891,19 +891,19 @@ public ChatServer()
 
   @Resource
   @Route("/users")
-  public Response.Content getUsers(String user, String token, String room, String filter, String dbName)
+  public Response.Content getUsers(String user, String token, String room, String filter, String limit, String dbName)
   {
     if (!tokenService.hasUserWithToken(user, token, dbName))
     {
       return Response.notFound("Petit malin !");
     }
 
-    List<UserBean> users;
-    if (room != null && !"".equals(room)) {
-      users = userService.getUsers(room, dbName);
-    } else {
-      users = userService.getUsers(filter, true, dbName);
-    }
+    int limit_ = 0;
+    try {
+      limit_ = Integer.valueOf(limit);
+    } catch (NumberFormatException e) {}
```

Avoid empty catch. In this case, the input parameters are not valid, so it should return an HTTP error 400 IMO.